src/Tgstation.Server.Api/Models/IrcConnectionStringBuilder.cs

```diff
@@ -88,6 +88,7 @@ public IrcConnectionStringBuilder(string connectionString)
 					case IrcPasswordType.NickServ:
 					case IrcPasswordType.Sasl:
 					case IrcPasswordType.Server:
+					case IrcPasswordType.Oper:
```

Comment:
This change needs documenting in `Nuget: API` and `HTTP API` 🆑 sections

Reply:
changed, did I do it correctly?